.cirrus/tasks.yml

```diff
@@ -233,24 +246,24 @@ trigger_release_task:
       - ./.cirrus/slack-notification.sh
 
 build_gcp_prod_app_task:
-  <<: *BUILD_GCP_STAGING_APP_TASK
+  <<: *BUILD_GCP_APP_TEMPLATE
   only_if: $TRIGGER == "PUSH_PROD_DEPLOYER"
   env:
-    GCLOUD_PRODUCT_NAME: sonarqube-dce-staging
+    GCLOUD_PRODUCT_NAME: official-sonarqube-data-center-edition
 
 verify_gcp_prod_app_task:
-  <<: *VERIFY_GCP_STAGING_APP_TASK
+  <<: *VERIFY_GCP_APP_TEMPLATE
   only_if: $TRIGGER == "PUSH_PROD_DEPLOYER"
   env:
-    GCLOUD_PRODUCT_NAME: sonarqube-dce-staging
+    GCLOUD_PRODUCT_NAME: official-sonarqube-data-center-edition
   depends_on:
     - build_gcp_prod_app
 
 release_gcp_prod_app_task:
-  <<: *BUILD_GCP_STAGING_APP_TASK
+  <<: *BUILD_GCP_APP_TEMPLATE
   only_if: $TRIGGER == "PUSH_PROD_DEPLOYER"
   env:
     GCLOUD_REGISTRY: gcr.io/sonarsource-public
     GCLOUD_PRODUCT_NAME: official-sonarqube-data-center-edition
   depends_on:
-    - verify_gcp_prod_app
\ No newline at end of file
+    - verify_gcp_prod_app
```

Comment:
If we make the changes I mentioned above, we should make this task dependent on verify_gcp_staging_app